plugins/ocm/README.md

```diff
@@ -1,10 +1,278 @@
-# Open Cluster Management
+# Open Cluster Management plugin for Backstage
 
-Welcome to the Open Cluster Management plugin!
+This plugin integrates your Backstage instance with Open Cluster Management's MultiClusterHub and MultiCluster Engine.
 
-_This plugin was created through the Backstage CLI_
+## Prerequisites
 
-## Getting started
+1. OCM deployed and configured on a kubernetes cluster
+2. Installed [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview)
+3. Kubernetes plugin is properly configured and able to connect to the Hub cluster for OCM via a `ServiceAccount` (the cluster is accessed by backstage backend, therefore it requires a permanent connection with constant RBAC scope)
+4. Following `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster:
+   ```yaml
+   kind: ClusterRole
+   apiVersion: rbac.authorization.k8s.io/v1
+   metadata:
+     name: backstage-ocm-plugin
+   rules:
+     - apiGroups:
+         - cluster.open-cluster-management.io
+       resources:
+         - managedclusters
+       verbs:
+         - get
+         - watch
+         - list
+   ```
+
+## Capabilities
+
+- All clusters represented as `ManagedCluster`s in MultiClusterHub/MCE are discovered and imported into Backstage catalog:
+  - Entity is defined as `kind: Resource` with `spec.type` set to `kubernetes-cluster`.
+  - Links to OCP console, OCM console and OpenShift Cluster Manager are provided in `metadata.links`.
+- Show real time data from OCM on Resource entity page including:
+  - Cluster current status (up, down, etc.)
+  - Cluster details (console link, OCP and Kubernetes version, etc.)
+  - Details about available compute resources on the cluster
+
+## Usage
+
+This plugin is made up of 2 packages:
+
+1. `@janus-idp/backstage-plugin-ocm-backend` connects backstage server to OCM.
+2. `@janus-idp/backstage-plugin-ocm` contains frontend components, requires `*-backend` to be present and set up.
```

Comment:
I think the `@janus-idp/backstage-plugin-ocm-common` should also be listed here since it contains the definition of the `ClusterDetails` object.

Reply:
Yea that makes sense.